arch/arc/core/fault_s.S

```diff
@@ -205,21 +205,23 @@ SECTION_SUBSEC_FUNC(TEXT,__fault,__ev_trap)
 	mov_s r6, K_SYSCALL_BAD
 
 valid_syscall_id:
+	_create_irq_stack_frame
+
 #ifdef CONFIG_ARC_SECURE_FIRMWARE
-	lr ilink, [_ARC_V2_ERSEC_STAT]
-	push ilink
+	/* ERSEC_STAT is IOW/RAZ in normal mode */
+	lr r0,[_ARC_V2_ERSEC_STAT]
```

Comment:
That's obviously a nitpicking but could you please keep spaces in between "," and the next symbol - it makes code much more readable. BTW I'm wondering if checkpatch checks assembly code or not? Have you run checkpatch on your changes?

Reply:
Fixed.

I ran checkpatch, it seems that it doesn't check assembly code